src/components/NewsLetter.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+const NewsLetter = () => {
+  return (
+    <div className='m-14 p-4 md:flex rounded-md shadow-md justify-around bg-primary'>
+
+      <div className='text-white flex flex-col md:flex-row justify-center items-center'>
+        <img className='w-24 md:w-40  h-100' src="https://www.freepnglogos.com/uploads/blue-email-logo-png-4.png" alt="email" />
+        <h1 className='text-[20px] sm:text-[40px] mb-3 sm:mb-0'>Subscribe to our <br /> Newsletter !</h1>
+      </div>
+
+      <div className='mt-4'>
+        <p className='text-white mb-5'>Subscribe to our newsletter and stay updated</p>
+        <div className='items-center sm:flex justify-center gap-2'>
+          <input type="text" placeholder='Write your email here' className='rounded p-1 w-full sm:w-fit md:p-2 mb-3 sm:mb-0 placeholder:text-primary border border-solid ' />
+          <button type='submit' className='md:p-2 p-1 w-full sm:w-fit rounded text-primary bg-tertiary'>Subscribe</button>
```

Comment:
I think you need to add Emailjs functionality. Let’s ask @Iseluin about this.

Reply:
@Iseluin I add Emaljs functionality?

Reply:
hey @MARWAHAKMI @OmarBassatni97 , nope, there isnt an issue about it yet since I didnt still decide on making this functional or not. Emailjs gives you a restricted amount of emails so, not yet.